haskell/BUILD.bazel

```diff
@@ -173,6 +179,21 @@ haskell_toolchain_info(
     name = "toolchain_info",
 )
 
+get_nodejs_toolchain(
+    name = "nodejs_toolchain",
+    visibility = ["//visibility:public"],
+)
+
+get_haskell_toolchain(
+    name = "haskell_toolchain",
+    visibility = ["//visibility:public"],
+)
+
+get_cc_toolchain(
+    name = "cc_toolchain",
+    visibility = ["//visibility:public"],
+)
```

Comment:
A common convention is to call these `current_XYZ_toolchain`. Might as well follow it here.